test/units/jwt_sessions/test_memory_store.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require 'jwt_sessions'
+
+class TestMemoryStore < Minitest::Test
+  attr_reader :store
+
+  def setup
+    @store = JWTSessions::StoreAdapters::MemoryStore.new
```

Comment:
You can set it back to redis on teardown, then there'll be no need to reset it back to redis on setup within other tests.

Reply:
Done, I like that idea!